app/controllers/sites_controller.rb

```diff
@@ -0,0 +1,30 @@
+class SitesController < ApplicationController
+  before_action :set_site
```

Comment:
We should update this controller to use `load_and_authorize_resource` (from cancan) instead of `set_site`.